calico-cloud/operations/cluster-management.mdx

```diff
@@ -8,9 +8,33 @@ description: Manage cluster storage, add new clusters, and configure alerts.
 
 To connect new clusters to {{prodname}}, select Managed Clusters in the left nav, and click **Connect Cluster**.
 
-## Recover a deleted managed cluster
+## Delete a managed cluster
 
-If you inadvertenty delete a managed cluster, rerun the `curl` command and the cluster will reconnect to {{prodname}}.
+### Prerequisites
+
+- Only users with either the `Owner` or `Admin` [roles](../users/user-management.mdx) are permitted to delete managed clusters.
+- The managed cluster must not be connected. For help with disconnecting your managed cluster, see [how to uninstall {{prodname}}](disconnect.mdx).
+
+### Delete a managed cluster
+
+1. In Manager UI, go to **Managed Clusters**.
+1. For the cluster you want to delete, select **Actions** > **Delete**.
+1. Verify details on the confirmation modal and click on **Delete** to confirm the managed cluster deletion.
+
+![delete managed cluster](/img/calico-cloud/managed-clusters-delete.png)
+![delete managed cluster modal](/img/calico-cloud/managed-clusters-delete-step1.png)
+
+:::note
+
+Deleting a managed cluster does not remove {{prodname}} logs.
+
+If a new managed cluster is connected with the same name as a previously deleted managed cluster, the logs associated with the deleted managed cluster will be displayed as if they belong to the new managed cluster.
+
+:::
+
+### Recover a deleted managed cluster
```

Comment:
I don't understand how this works. If we've removed the cluster from the list, how do we find it again?

Reply:
The procedure is incorrect, as you mentioned the cluster will be removed from the list.
The correct procedure is to connect the cluster (with the same name as the previously deleted) https://github.com/tigera/docs/pull/961/commits/877124c80c729c5ae25d3a24696e88e08a885ea6